cases/services/tag_enricher.py

```
@@ -0,0 +1,1173 @@
+    md_converter = MarkItDown(enable_plugins=True)
+
+    files_to_try = []
+    if src.uploaded_file and hasattr(src.uploaded_file, "path"):
+        files_to_try.append(src.uploaded_file.path)
+
+    try:
+        uploads = DocumentSourceUpload.objects.filter(source_id=src.id)
+        for upload in uploads:
+            if upload.file and hasattr(upload.file, "path"):
+                files_to_try.append(upload.file.path)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)